modules/doc/content/newsletter/2020_11.md

```diff
@@ -14,14 +14,48 @@ review see the [Reporters/index.md].
 
 ## Unit Conversion in Input Files
 
-A physical [units parsing and conversion](utils/Units.md) helper system was added to MOOSE. A unit object can be constructed form a string (e.g. `N*m`, `eV/at`, `kg*(m/s)^2`, `1/s`, `m^-3`). The unit parser supports the `*`,`/`, and `^` operators as well as parenthesis `(`, `)` to formulate composit units. Metric prefixes are supported (e.g. `MPa`, `kg`, `mm`). 
+A physical [units parsing and conversion](utils/Units.md) helper system was added to MOOSE. A unit object can be constructed form a string (e.g. `N*m`, `eV/at`, `kg*(m/s)^2`, `1/s`, `m^-3`). The unit parser supports the `*`,`/`, and `^` operators as well as parenthesis `(`, `)` to formulate composit units. Metric prefixes are supported (e.g. `MPa`, `kg`, `mm`).
```

Comment:
```suggestion
A physical [units parsing and conversion](utils/Units.md) helper system was added to MOOSE. A unit object can be constructed form a string (e.g. `N*m`, `eV/at`, `kg*(m/s)^2`, `1/s`, `m^-3`). The unit parser supports the `*`,`/`, and `^` operators as well as parenthesis `(`, `)` to formulate composite units. Metric prefixes are supported (e.g. `MPa`, `kg`, `mm`).
```
Technically, if you support ^ (not unit exponentials, just integers) you also support -
Just a detail though, I have no opinion